website/docs/cloud-docs/users-teams-organizations/2fa.mdx

```diff
@@ -12,7 +12,11 @@ User accounts can be additionally protected with two-factor authentication (2FA)
 
 ## Setting up Two-factor Authentication
 
-To reach your user security settings page, click the user icon in the upper right corner and choose **Account Settings** from the menu.
+To reach your user security settings page:
+
+1. Sign in to [HCP Terraform](https://app.terraform.io/).
```

Comment:
Does TFE support 2FA?

Reply:
Oh, they do! Thank you, I forgot to update this one!
```suggestion
1. Sign in to [HCP Terraform](https://app.terraform.io/) or Terraform Enterprise.
```